src/server/models/cacheable_queries/campaign-contact.js

```diff
@@ -562,6 +562,58 @@ const campaignContactCache = {
     } catch (err) {
       console.log("contact updateStatus Error", newStatus, contact, err);
     }
+  },
+  updateCustomFields: async (contact, customFields, campaign, organization) => {
```

Comment:
This is the part of the code that I understand the least because of my knowledge of caching. I'm trying to think of if there are any particularly clever ways to test this...